DEPLOYMENT-CHECKLIST.md

```diff
@@ -0,0 +1,71 @@
+# Deployment Checklist
+
+Use this checklist to validate a successful Guardian Connector deployment before handoff.  
+It covers the manual steps that are not already automated through the scripts and workflows in this repo, and is meant to complement — not replace — the deployment guides.
+
+## VM
+
+- [ ] Did I set a `*.alias` A record in my DNS provider? (And can confirm it resolves to the VM's IP?)
+- [ ] Did I set an appropriate file quota (e.g. 1TB) for file shares?
+
+## CapRover and services
+
+### CapRover
+
+- [ ] Did I set up the Guardian Connector 3rd party repository?
+- [ ] Did I deploy all required apps?
+- [ ] Did I configure environment variables and settings for each app correctly? (c.f. [caprover/INSTALL_GC_STACK.md#post-install-app-configuration](caprover/INSTALL_GC_STACK.md#post-install-app-configuration))
+- [ ] Did I set the `{alias}.domain.net` to point to one of the applications (e.g. Superset or GC Landing Page)?
+
+### Superset
+
+- [ ] Did I set a Mapbox API key and a logo in the environment variables?
+- [ ] Did I copy down the `SECRET_KEY` from the environment variables and store it in Keepass?
+- [ ] Did I successfully log in as the initial admin account using auth0?
+- [ ] Did I configure a database connection to `warehouse`?
+
+### Filebrowser
+
+- [ ] Did I get the Filebrowser admin password from the logs in CapRover, and change it?
+- [ ] Did I store the updated admin credentials in Keepass?
+
+### Windmill
+
+- [ ] Did I set up auth0 for Windmill?
+- [ ] Did I create a Windmill workspace?
+- [ ] Did I push `gc-scripts-hub` content using the Windmill CLI (`wmill sync push`)?
+- [ ] Did I create the standard Windmill resources (e.g. for alerts)?
+  - [ ] Postgres database
+  - [ ] Twilio message template
+  - [ ] GFW API key
+  - [ ] GCP service account
+  - [ ] CoMapeo archive server
+- [ ] Did I invite other required admin users to the Windmill instance and workspace?
+
+### CoMapeo
+
+- [ ] Did I set an appropriate project limit for the CoMapeo archive server in the `ALLOWED_PROJECTS` environment variable?
+
+### GC Explorer
+
+_No manual steps required until datasets exist._
+
+## Third-party services
+
+###Auth0
+
+- [ ] Did I add a GCP OAuth client for Google social login?
+- [ ] Did I set up the user approval flow?
+- [ ] Is RBAC set up?
+- [ ] Did I assign the Admin role to my logged in user?
+- [ ] Did I invite other required admins to the Auth0 tenant?
+- [ ] Did I create applications for each GC Stack app?
```

Comment:
This has to happen before `CapRover > Did I deploy all required apps`.  Dunno if the list is supposed to be in-order or just an audit.

Given what you wrote about "not intended to replace guides" then I think it's fine to be out-of-order.  In that case maybe the whole document (or at least the header and the link from README.md) ought to be called "Post-deployment checklist"